### ISSUE TITLE
Training issue!!!!

### ISSUE THREAD

**Issue by zero-rong** [Vill-Lab/2023-TIFS-ISTVT#3]
I use the XceptionVidTr model (release code) to train on the FF++, and validate the model on CDF2. However, I found that my model was overfitting. In previous rounds of training, the AUC of the validation set (CDF2) decreased as the number of rounds increased. What's the reason, is there any similar problem like me.
